.github/workflows/docker-bases.yml

```diff
@@ -229,10 +236,10 @@ jobs:
           platforms: ${{ matrix.platform }}
           build-args: |
             arch=nvc
-            ver=nvhpc-25-7
+            ver=${{ matrix.nvhpc_version }}
```

Comment:
Isn't this gonna error for the arm build that doesn't have a version?

Reply:
According to the docs a versioned version should exist

```bash
curl https://developer.download.nvidia.com/hpc-sdk/ubuntu/DEB-GPG-KEY-NVIDIA-HPC-SDK | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-hpcsdk-archive-keyring.gpg
echo 'deb [signed-by=/usr/share/keyrings/nvidia-hpcsdk-archive-keyring.gpg] https://developer.download.nvidia.com/hpc-sdk/ubuntu/arm64 /' | sudo tee /etc/apt/sources.list.d/nvhpc.list 
sudo apt-get update -y
sudo apt-get install -y nvhpc-25-7 
```

[ref](https://developer.nvidia.com/nvidia-hpc-sdk-257-downloads)

Reply:
or is it a cross product. Not sure, guess should just trigger it and see